I have looked at the failing tests.

Apparently the performance-8.0 test is not prepared for PRs from cloned repositories.

> Run git fetch origin dav_read_share_notes
> fatal: couldn't find remote ref dav_read_share_notes
> Error: Process completed with exit code 128.

Regarding acceptance tests - the errors are suspiciously close to my changes (both relate to the sharing feature). Unfortunately I have not been able to correlate the reported errors with potential errors in the backend. Are logs from the backend available somewhere for the failed run?

--- a/apps/dav/lib/Connector/Sabre/Node.php
+++ b/apps/dav/lib/Connector/Sabre/Node.php
@@ -365,7 +365,7 @@ abstract class Node implements \Sabre\DAV\INode {
 
 		$share = $storage->getShare();
 		$note = $share->getNote();
-		if ($share->getShareOwner() !== $user && !empty($note)) {
+		if ($share->getShareOwner() !== $user) {
 			return $note;
 		}
 		return '';

